portal-web/test/functional/com/liferay/portalweb/macros/SAMLNavigation.macro

```diff
@@ -20,4 +20,10 @@ definition {
 		Navigator.gotoNavTab(navTab = "Service Provider Connections");
 	}
 
+	macro viewConsoleValue {
+		Pause(locator1 = "5000");
```

Comment:
I think we could create some macro, which waits dynamically for a limited time to see the text in the console. I'm not sure if we have a similar macro already, or not.